.cargo/config.toml

```diff
@@ -5,3 +5,7 @@ protocol = "sparse"
 # https://github.com/rust-lang/cargo/issues/4133
 [target.aarch64-unknown-linux-gnu]
 linker = "aarch64-linux-gnu-gcc"
+
+[profile.profiling]
```

Comment:
why do we need this -> for profiling we might not need such a profile, it really depends on the profiling tool which is used. lets not have this profile called profiling until we know what profiler tool we want to inject (flamegraphe etc.)

Reply:
understood. closing.